src/main/java/mate/academy/spring/boot/service/BookServiceImpl.java

```diff
@@ -63,10 +63,17 @@ public void deleteById(Long id) {
     @Override
     @Transactional
     public BookDto updateBook(Long id, CreateBookRequestDto requestDto) {
-        Book updateBook = bookMapper.updateBookFromDto(
-                bookRepository.findById(id).orElseThrow(() ->
-                        new EntityNotFoundException("Can't get book by id = " + id)), requestDto);
-        return bookMapper.toDto(updateBook);
+        String isbn = null;
+        Book oldBook = bookRepository.findById(id).orElseThrow(() ->
+                new EntityNotFoundException("Can't get book by id = " + id));
+        if (requestDto.isbn() == null || requestDto.isbn().isBlank()) {
```

Comment:
Does CreateBookRequestDto have some validations for isbn field like NotBlank? 
Also, your Book entity should has constraint for isbn like nullable = false and unique = true
Check these please, and remove redundant logic if needed